classes/core/PKPApplication.php

```diff
@@ -294,7 +294,16 @@ public static function get()
      */
     public function getUUID(): string
     {
-        return $this->getRequest()->getSite()->getUniqueSiteID();
+        $site = $this->getRequest()->getSite();
+        $uniqueSiteId = $site->getUniqueSiteID();
+        if (is_null($uniqueSiteId) || strlen($uniqueSiteId) == 0) {
```

Comment:
Another way could be `if (!strlen((string) $uniqueSiteId)))`, but that's just my personal preference :)